linodecli/baked/request.py

```diff
@@ -44,11 +46,16 @@ def __init__(
         #: the larger response model
         self.path = prefix + "." + name if prefix else name
 
-        #: The description of this argument, for help display
-        self.description = (
-            schema.description.split(".")[0] if schema.description else ""
+        description_rich, description = process_arg_description(
+            schema.description or ""
         )
 
+        #: The description of this argument for Markdown/plaintext display
+        self.description = description
+
+        #: The description of this argument for display on the help page
+        self.description_rich = description_rich
```

Comment:
We're now baking the rendered descriptions directly into the request for re-use and optimization purposes. 

NOTE: The `description` field is preserved here in preparation for potential docs generation down the road.